composables/use-tasks-fetch.ts

```diff
@@ -121,19 +104,58 @@ export function useTasksFetch() {
   const {
     data: tasks,
     pending: areTasksPending,
-    refresh: refreshTasks,
+    refresh: refetchAllTasks,
   } = useAsyncData('tasks', async () => {
-    const issuesAndPatches = await Promise.all([fetchIssues(), fetchPatches()])
+    const issuesAndPatches = await Promise.all([fetchAllIssues(), fetchAllPatches()])
     const tasks = issuesAndPatches.flat()
 
     return tasks
   })
 
+  async function refetchTask(task: Task): Promise<void> {
+    if (!tasks.value) {
+      return
+    }
+
+    let refetchedTask: Task
+    switch (task.rpb.kind) {
+      case 'issue':
+        refetchedTask = await fetchIssueById(task.id)
+        break
+      case 'patch':
+        refetchedTask = await fetchPatchById(task.id)
+        break
+      default:
+        throw new Error('Unsupported task kind')
```

Comment:
In these cases, it's best to always use `assertUnreachable()` util (just copy-paste it from [here](https://app.radicle.at/nodes/seed.radicle.at/rad:z3Makm6fsQQXmpSFE43DZqwupaEhk/tree/src/utils/assertions.ts#L24)) to ensure that when typings change in the future all related errors will be caught at compile time instead of runtime. Makes refactorings a joy too.